idea/tests/testData/intentions/addNameToArgument/mixingPositionedAndNamedDisabled.kt

```diff
@@ -0,0 +1,8 @@
+// IS_APPLICABLE: false
+// COMPILER_ARGUMENTS: -XXLanguage:-MixedNamedArgumentsInTheirOwnPosition
```

Comment:
@darthorimar I could not figure out why the FE 1.0 intention tests are ignoring these language settings. Right now it fails. Even `// LANGUAGE_VERSION: 1.3` gets ignored; in a debugger I can see the version is 1.5.

Reply:
Probably, this is because `AbstractIntentionTest` does not support such directives. You may override`getProjectDescriptor` in `AbstractIntentionTest` and add needed behavior there

Reply:
That does not seem to be the cause. The directives are being read in the call to [`withCustomCompilerOptions()`](https://github.com/JetBrains/intellij-kotlin/blob/09b296ff5b52d77e5b66c87b3a509c6303cad9b4/idea/tests/test/org/jetbrains/kotlin/idea/intentions/AbstractIntentionTest.kt#L101). The problem is the preceding call to `myFixture.configureByFiles()` actually causes resolution on FE 1.0 but not on FIR. The resolution causes the language version settings for the project and module to be cached, and this happens before the custom compiler options were read. Moving that `configureByFiles()` to inside `withCustomCompilerOptions()` fixes the issue (see latest commit).

As to why resolution happens for FE 1.0, I dug into it a little bit and the resolution happens here [in `KotlinLanguageInjectionContributor`](https://github.com/JetBrains/intellij-kotlin/blob/09b296ff5b52d77e5b66c87b3a509c6303cad9b4/jvm/src/org/jetbrains/kotlin/idea/injection/KotlinLanguageInjectionContributor.kt#L277) at some point during the fixture setup. Specifically, I think it happens because the caret is inside a string literal and some kind of check happens for injection contributors. `KotlinLanguageInjectionContributor` is only present in the FE 1.0 plugin.